docs/docs/configuration/index.md

```diff
@@ -435,4 +435,14 @@ cameras:
       quality: 70
       # Optional: Restrict mqtt messages to objects that entered any of the listed zones (default: no required zones)
       required_zones: []
+
+    # Optional: Configuration for how camera is handled in the GUI.
+    ui:
+      # Optional: Adjust sort order of cameras in the UI. Larger numbers come later (default: shown below)
+      # By default the cameras are sorted alphabetically.
+      order: 0
+      # Optional: Whether or not to show the camera in the Frigate UI (default: shown below)
+      dashboard: True
+      # Optional: Whether or not to show the camera in the Frigate Birdseye View (default: shown below)
+      birdseye: True
```

Comment:
I wouldn't consider this to be a `ui` setting. Perhaps we should have a camera level birdseye config instead. This scales better for future birdseye config sections. We could have mode configurable by camera and allow order to be set for cameras too. 

```yaml
birdseye:
  enabled: True
```